src/test/java/core/basesyntax/ReturnHandlerTest.java

```diff
@@ -0,0 +1,90 @@
+package core.basesyntax;

```

Comment:
Your test class should be in the same package as the class that it tests. Fix it for all tests 
also. to create a test you can use the power of IDE https://www.jetbrains.com/help/idea/create-tests.html